### PR TITLE
feat(atoms/button): add button

### PR DESCRIPTION
**Pull request by marionebl**
~~WIP, don't merge yet~~
- Adds a basic button component
- `primary`, `secondary` orders
- `inverted` flag
- Text only

### CODE REVIEW

**Comment by Dangoo** [comment]
Is a `button` really an atom if it can host an `icon`?

**Comment by marionebl** [comment]
I think the newest version reflects this problem by removing the icon capabilities from `atoms/button`. The current plan is to have a `molecules/icon-button` if needed instead

**Comment by marionebl** [comment]
Ready for review @Dangoo

File: patterns/atoms/button/demo.css [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-	width: 100%;
-	border-collapse: collapse;
+.pp-demo {
+	display: flex;
+	padding: 20px;
 
-	td, th {
-		text-align: left;
-		padding: 8px 12px;
+	@nest &__container {
```
Comment (Dangoo) [inline patterns/atoms/button/demo.css]: Simple selectors can be [nest-prefixed](https://tabatkins.github.io/specs/css-nesting/#nest-prefixed) where @nest can be omitted

**Comment by Dangoo** [comment]
I really like the hover effect for `button button--primary button--inverted` but the effect should be inverted. So white background as default becoming transparent on hover. Otherwise primary and secondary buttons are not distinguishable from each other.

**Pull request merged**